Black Clovedle/Guess.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  MatchupsTableViewCell.swift
+//  Black Clovedle
+//
+//  Created by Scott DiBenedetto on 8/7/24.
+//
+
+import UIKit
+
+class Guess: UITableViewCell {
+
+    @IBOutlet weak var characterDebutArch: UIImageView?
+    @IBOutlet weak var characterSpirit: UIImageView?
+    @IBOutlet weak var characterMAffiliation: UIImageView?
+    @IBOutlet weak var characterAffiliation: UIImageView?
+    @IBOutlet weak var characterGender: UIImageView?
+    @IBOutlet weak var characterText: UILabel?
+    @IBOutlet weak var characterImage: UIImageView?
+    
+    @IBOutlet weak var debutArchText: UILabel!
+    @IBOutlet weak var spiritText: UILabel!
+    @IBOutlet weak var magicAffiliationText: UILabel!
+    @IBOutlet weak var affiliationText: UILabel!
+    @IBOutlet weak var genderText: UILabel!
+    var greenCorrect: UIImage = UIImage(named: "greenCorrect")!
```

Comment:
Idk if this will work as is, but for image initialization there's a way to have xcode automatically maintain an enum called `ImageResource` where you can do `UIImage(resource: .greenCorrect)` and it returns a non-optional UIImage. I would prefer that to using "named" because if you, for example, remove this image for whatever reason, the compiler won't complain but the app will crash. With the resource stuff, if you remove the image, the app won't compile and you'll have to find a way to fix it, which is preferable